### PR TITLE
Feat/like page

### PR DESCRIPTION
**Pull request by manish591**
#### Added Like page functionality

### CODE REVIEW

File: src/components/card/HorizontalCard.jsx [hunk 1]
```diff
@@ -2,18 +2,18 @@ import React from "react";
 
 const HorizontalCard = () => {
   return (
-    <section class="card card--horizontal" style={{ inlineSize: "100%" }}>
-      <div class="card__image-container" style={{ objectFit: "contain" }}>
+    <section className="card card--horizontal" style={{ inlineSize: "100%" }}>
+      <div className="card__image-container" style={{ objectFit: "contain" }}>
         <img
           src="https://i.ytimg.com/vi/inVClSNYBQ4/hq720.jpg?sqp=-oaymwEcCNAFEJQDSFXyq4qpAw4IARUAAIhCGAFwAcABBg==&rs=AOn4CLD1dg5JXDycuG06NEn9A-0Pnd40zQ"
           alt=""
```
Comment (OmKakatkar) [inline src/components/card/HorizontalCard.jsx:9]: Try not to keep alt attribute empty

**Review by OmKakatkar — APPROVED** [review]
Like:
* Use of a custom hook for api requests that was new for me.
* Good variable and class names

**Review by romabulani — APPROVED** [review]
As Om already suggested give value to alt attribute, rest all looks good to merge

**Comment by manish591** [comment]
> As Om already suggested give value to alt attribute, rest all looks good to merge

This is temporary image I'll add another image and then I'll add alt

**Pull request merged**